porcelain/miner.go

```diff
@@ -295,6 +298,57 @@ func MinerGetOwnerAddress(ctx context.Context, plumbing mgoaAPI, minerAddr addre
 	return address.NewFromBytes(res[0])
 }
 
+// MinerGetSectorSize queries for the sector size of the given miner.
+func MinerGetSectorSize(ctx context.Context, plumbing mgoaAPI, minerAddr address.Address) (*types.BytesAmount, error) {
+	rets, err := plumbing.MessageQuery(ctx, address.Address{}, minerAddr, "getSectorSize")
+	if err != nil {
+		return nil, errors.Wrap(err, "'getSectorSize' query message failed")
+	}
+
+	methodSignature, err := plumbing.ActorGetSignature(ctx, minerAddr, "getSectorSize")
+	if err != nil {
+		return nil, errors.Wrap(err, "failed to acquire 'getSectorSize' signature")
+	}
+
+	sectorSizeVal, err := abi.Deserialize(rets[0], methodSignature.Return[0])
+	if err != nil {
+		return nil, errors.Wrap(err, "failed to deserialize returned value")
+	}
+
+	sectorSize, ok := sectorSizeVal.Val.(*types.BytesAmount)
+	if !ok {
+		return nil, errors.New("failed to convert returned ABI value")
+	}
+
+	return sectorSize, nil
+}
+
+// MinerGetLastCommittedSectorID queries for the id of the last sector committed
+// by the given miner.
+func MinerGetLastCommittedSectorID(ctx context.Context, plumbing mgoaAPI, minerAddr address.Address) (uint64, error) {
```

Comment:
Not sure I agree with it, but some arguments have been made in the past that nothing should be exposed in porcelain/plumbing that isn't explicitly useful to the end user. I think an exception is warranted here, but I'm interested to hear @acruikshank's opinion.

Reply:
Actually, on second thought, since this is only used in `node/node.go`, I'm inclined to say it should just live there.

Reply:
We are trying to move all the code out of node. When the code that depends on this functionality moves, it will move into protocol. Helper functionality to support protocols is an appropriate use of porcelain, so I think this is a great refactor.